docs/api-reference/type-aliases/WorkerEventHandler.mdx

```diff
@@ -0,0 +1,83 @@
+[testplane](../README.mdx) / WorkerEventHandler
+
+# Type Alias: WorkerEventHandler()\<T\>
```

Comment:
Needs fix: Something very weird is going on here. This page should either be deleted or fixed